spec/slack/web/faraday/request_spec.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+require 'spec_helper'
+
+class DummyClient
```

Comment:
Move this inside `describe` to be a subclass?

Reply:
I know rubocop tends complains about that style with https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Lint/ConstantDefinitionInBlock so I typically define it outside, but if that's your preference I can move it inside. Another option is I can give the class a much more descriptive name to reduce the chances of a conflicting class being defined elsewhere

Reply:
Not important to me.